src/components/RichTextEditor/context/RichTextEditorContext.ts

```diff
@@ -0,0 +1,22 @@
+import { createContext, useContext } from 'react';
+import { DesignTokens } from '../types';
+import { defaultDesignTokens } from '../utils/defaultDesignTokens';
+import { EditorPositioningWrapper, Position } from '../EditorPositioningWrapper';
+
+type RichTextEditorContextProps = {
+    designTokens: DesignTokens;
+    PositioningWrapper: typeof EditorPositioningWrapper[Position.FLOATING];
```

Comment:
IMHO only the position of the toolbar should be provided by the context, since the component needs to rerender on every context change